Northstar.CustomServers/mod/scripts/vscripts/mp/_stats.nut

```diff
@@ -1035,7 +1035,11 @@ void function SaveStatsPeriodically_Threaded()
 	while( true )
 	{
 		foreach( entity player in GetPlayerArray() )
+		{
+			if ( !IsValid( player ) )
```

Comment:
there is a space too much after if

Reply:
No there isn't?

Reply:
forgot that we allow both with and without that space